examples/run_nutation_amp_scopecapture.py

```diff
@@ -0,0 +1,116 @@
+from pyspecdata import figlist_var, getDATADIR, ndshape
+import os
+import SpinCore_pp
+from datetime import datetime
+import numpy as np
+import h5py
+from Instruments.XEPR_eth import xepr
+from Instruments.gds import GDS_scope
+from SpinCore_pp.ppg import run_spin_echo
```

Comment:
I don't know how this guy got into this PR should I remove and make a different PR for this? I honestly don't remember what this guy does

Reply:
it might be the run_nutation_amplitude.py renamed?